src/features/offer/components/OfferAbout/OfferAbout.tsx

```diff
@@ -35,7 +35,7 @@ export const OfferAbout: FunctionComponent<Props> = ({
           {offer.description ? (
             <View>
               <TypoDS.BodyAccent>Description&nbsp;:</TypoDS.BodyAccent>
-              <CollapsibleText numberOfLines={NUMBER_OF_LINES_OF_DESCRIPTION_SECTION}>
+              <CollapsibleText numberOfLines={NUMBER_OF_LINES_OF_DESCRIPTION_SECTION} isMarkdown>
                 {offer.description}
               </CollapsibleText>
```

Comment:
je trouve ça bizarre de devoir ajouter une props assez spécifique : la gestion du markdown à un composant générique

je propose de garder le composant générique et d'avoir un composant qui s'occupe de transformer des children en markdown

```suggestion
              <CollapsibleText numberOfLines={NUMBER_OF_LINES_OF_DESCRIPTION_SECTION}>
                <Markdown>{offer.description}</Markdown>
              </CollapsibleText>
```